src/parse/parse_template.rs

```diff
@@ -44,6 +44,7 @@ pub fn parse_template(
         Some("]"),
       ));
     },
+    "{" => { return Template::substack(parse_program_sequence(source_iter, Some("}"))); },  // TODO Make correct type
```

Comment:
Like is there a type for this? idk, but the parsing gets funky if we ignore opening and closing stuff

Reply:
I don't know what I meant when I wrote that. It is probably outdated, since I went quite wild on the typing recently. Looks good to me as-is (:

Reply:
I thought {0 "hello"} meant creating like a ananomus type with {int, string} or something. At last something different from (0 "hello") which was a substack. Not that I 100% remember what that was, but pretty sure it wasn't a type

Reply:
Right! Yes, that should be a Set not a Substack!

But also, I'm probably dropping support for creating Sets directly with a literal in order to make it easier to parse creating a Struct och Dict with literal syntax, so it is kinda moot and will be fixed soon-ish.